db/data/20220519162059_fix_primary_pe_subjects.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class FixPrimaryPeSubjects < ActiveRecord::Migration[6.1]
+  def up
+    trainee_ids = [
+      100790,
+      90738,
+      90740,
+      90747,
+      91105,
+      102185,
+      106488,
+      106494,
+      106564,
+      106573,
+      106584,
+      106539,
+      106590,
+      106595,
+      106572,
+      106568,
+      106658,
+      106650,
+      106676,
+      106695,
+      106697,
+      106714,
+      106735,
+      106778,
+      106781,
+      106774,
+      109263,
+      109265,
+      85174,
+      109624
+    ]
+
+    Trainee.where(id: trainee_ids).each do |trainee|
+      trainee.without_auditing do
+        trainee.update(course_subject_one: "primary teaching", course_subject_two: "sport and exercise sciences", course_allocation_subject_id: 21)
```

Comment:
Can we rely on `course_allocation_subject_id` being 21 or do we need to lookup the value by name?